addon-resizer/nanny/nanny_lib_test.go

```diff
@@ -232,34 +232,35 @@ func TestUpdateResources(t *testing.T) {
 	oneMinuteAgo := now.Add(-time.Minute)
 	oneHourAgo := now.Add(-time.Hour)
 	testCases := []struct {
-		th   uint64
-		x, y corev1.ResourceList
-		lc   time.Time
-		sud  time.Duration
-		sdd  time.Duration
-		want updateResult
+		th          uint64
+		x, y        corev1.ResourceList
+		lc          time.Time
+		sud         time.Duration
+		sdd         time.Duration
+		scalingMode string
+		want        updateResult
 	}{
 		// No changes to the resources
-		{0, standard, standard, now, noDelay, noDelay, noChange},
-		{0, standard, standard, oneHourAgo, noDelay, noDelay, noChange},
-		{0, standard, standard, oneHourAgo, oneMinuteDelay, noDelay, noChange},
-		{0, standard, standard, oneHourAgo, noDelay, oneMinuteDelay, noChange},
-		{10, standard, siStandard, now, noDelay, noDelay, noChange},
+		{0, standard, standard, now, noDelay, noDelay, NodeProportional, noChange},
```

Comment:
Please keep existing tests for node-proportional and add new ones for container-proportional

Reply:
Addressed.